readme/7.0/BREAKING_CHANGES.markdown

```diff
@@ -4344,6 +4344,56 @@ configuration changes.
 
 ---------------------------------------
 
+### Moved Captcha Portal Properties to OSGi Configuration
+- **Date:** 2017-Feb-13
+- **JIRA Ticket:** LPS-67830
+
+#### What changed?
+
+The captcha properties have been moved from portal.properties and Server
+Administration to an OSGi configuration named
+`CaptchaConfiguration.java` in the `captcha-api` module.
+
+#### Who is affected?
+
+This affects anyone who is using the following portal properties:
+
+- `captcha.max.challenges`
+- `captcha.check.portal.create_account`
+- `captcha.check.portal.send_password`
+- `captcha.check.portlet.message_boards.edit_category`
+- `captcha.check.portlet.message_boards.edit_message`
+- `captcha.engine.impl`
+- `captcha.engine.recaptcha.key.private`
+- `captcha.engine.recaptcha.key.public`
+- `captcha.engine.recaptcha.url.script`
+- `captcha.engine.recaptcha.url.noscript`
+- `captcha.engine.recaptcha.url.verify`
+- `captcha.engine.simplecaptcha.height`
+- `captcha.engine.simplecaptcha.width`
+- `captcha.engine.simplecaptcha.background.producers`
+- `captcha.engine.simplecaptcha.gimpy.renderers`
+- `captcha.engine.simplecaptcha.noise.producers`
+- `captcha.engine.simplecaptcha.text.producers`
+- `captcha.engine.simplecaptcha.word.renderers`
+
+#### How should I update my code?
+
+Instead of overriding the `portal.properties` file, you can manage the
+properties from Portal's configuration administrator. This can be accessed by
+navigating to Liferay's *Control Panel* &rarr; *Configuration* &rarr; *System
+Settings* &rarr; *Captcha* and editing the settings there.
+
+If you would like to include the new configuration in your application, follow
+the instructions for
+[making your applications configurable in Liferay 7.0](https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/making-your-applications-configurable).
+
+#### Why was this change made?
+
+This change was made as part of the modularization efforts to ease portal
+configuration changes.
+
+---------------------------------------
```

Comment:
Brian moved this to `readme/7.1/BREAKING_CHANGES.markdown` for my User File Uploads pull.